packages/next/build/index.ts

```diff
@@ -950,6 +957,12 @@ export default async function build(
       const buildManifest = JSON.parse(
         await promises.readFile(buildManifestPath, 'utf8')
       ) as BuildManifest
+      const appBuildManifest =
+        appDir && existsSync(appBuildManifestPath)
```

Comment:
```suggestion
        appDir
```
Should the `appBuildManifest` always exist if `appDir` is enabled? If so seems like we would want to error if it was missing.